### ISSUE TITLE
Starting from terminal does not load theme

### ISSUE THREAD

**Issue by kevincolyar** [syl20bnr/spacemacs#8633]
#### Description :octocat:
Starting spacemacs from the terminal does not load the default theme (spacemacs-dark). However, running load-theme and selecting the same theme successfully loads the theme.  No issue when running GUI version of emacs.

#### Reproduction guide :beetle:
- Start Emacs from terminal.

*Observed behaviour:* :eyes: :broken_heart:
Theme is not the one specified in `.spacemacs` file.

*Expected behaviour:* :heart: :smile:
Theme should be the one specified in `.spacemacs` file.

#### System Info :computer:
- OS: darwin
- Emacs: 25.1.1
- Spacemacs: 0.200.8
- Spacemacs branch: master (rev. bd7ef98e)
- Graphic display: nil
- Distribution: spacemacs
- Editing style: vim
- Completion: helm
- Layers:
```elisp
(helm auto-completion better-defaults emacs-lisp git markdown org spell-checking syntax-checking
      (version-control :variables version-control-diff-tool 'diff-hl)
      ruby ruby-on-rails osx clojure html javascript ranger yaml python)
```
- System configuration features: JPEG RSVG NOTIFY ACL GNUTLS LIBXML2 ZLIB TOOLKIT_SCROLL_BARS NS MODULES

**Comment by asnider6** [comment]
I'm also experiencing this, attempting to use `base16-ocean` (`base16-theme` is being added in `additional packages`)

**Comment by aruiz14** [comment]
I also experience the issue when using a theme installed via additional packages. If the theme is from the megapack it works normally. The `cycle-spacemacs-theme` `SPC T n` is not working though, which may be related.
Using `helm-themes` `SPC T s` works.

**Comment by hallabro** [comment]
Same here. Loading it manually works fine (after being asked to trust the theme etc) but automatic loading does not.

**Comment by asnider6** [comment]
On a tangentially related note, base16 themes no longer are working correctly in the terminal. Previous to 200.8, I was able to use it with base16-shell and a few background configurations. The update has borked the colors namely on the current line highlighting and mode line (syntax colors are working correctly)

**Comment by shanesveller** [comment]
Based on `git bisect`, I experience this behavior starting with Spacemacs commit `11d7bb526c02346bcd7fcac5121ff51d3805b1e2`.

**Comment by syl20bnr** [comment]
Sorry for the regression and thank you for the report and investigation 💜 
I will fix it later today.

**Comment by wang0z** [comment]
For your reference.

iff --git a/core/core-themes-support.el b/core/core-themes-support.el
index a8ba954..b87ea8e 100644
--- a/core/core-themes-support.el
+++ b/core/core-themes-support.el
@@ -222,8 +222,10 @@ package name does not match theme name + `-theme' suffix.")
         ;; Apply theme
         (mapc 'disable-theme custom-enabled-themes)
         ;; explicitly reload the theme for the first GUI client
-        (eval `(spacemacs|do-after-display-system-init
-                (load-theme ',theme t)))
+        (if (display-graphic-p)
+            (eval `(spacemacs|do-after-display-system-init
+                    (load-theme ',theme t)))
+          (load-theme theme t))
         (when install
           (spacemacs-buffer/replace-last-line
            (format (concat "--> User theme \"%s\" has been applied, you may "

**Comment by apenney** [comment]
Any news on this one?  Feeling down in the dumps with my half broken lookin' spacemacs!

**Comment by solnic** [comment]
Is it somehow related to line numbers missing? I have `global-linum-mode t` in user-config and it has no effect. The only way to enable line numbers is to use `spacemacs/toggle-line-numbers-on`.

**Comment by hallabro** [comment]
@solnic I had the same issue when upgrading but why would it be related? I set `dotspacemacs-line-numbers 'relative` and it works great for me.

Please open a new issue instead if the above doesn't work.

**Comment by syl20bnr** [comment]
Fixed in 0.200.9 , thanks to @wang0z !
